Shared/BetterColorPicker.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using BepInEx;
+using BepInEx.Configuration;
+using ChaCustom;
+using HarmonyLib;
+using Illusion.Component.UI.ColorPicker;
+using IllusionUtility.GetUtility;
+using KKAPI.Utilities;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+using PluginInfo = BetterColorPicker.Info;
+
+namespace BetterColorPicker
+{
+    [BepInDependency(KKAPI.KoikatuAPI.GUID)]
+    [BepInPlugin(PluginInfo.GUID, "Better Color Picker", PluginInfo.Version)]
+    [BepInProcess(Constants.MainGameProcessName)]
+    public class BetterColorPicker : BaseUnityPlugin
+    {
+        //public const string GUID = "marco.better_color_picker";
+        //public const string Version = "2.0.2";
+
+        private const string BtnText = "Pick color from desktop";
+        private const string BtnTextActive = "* Press any key to finish *";
+
+        private static Texture2D _lut;
+        private static Action<Color> _setColor;
+        private static TextMeshProUGUI _buttonText;
+
+        private static bool _capturing;
+        private static bool Capturing
+        {
+            get => _capturing;
+            set
+            {
+                _capturing = value;
+                if (_buttonText != null)
+                    _buttonText.text = _capturing ? BtnTextActive : BtnText;
+            }
+        }
+
+        public ConfigEntry<bool> ColorAdjust { get; private set; }
+
+        private void Awake()
+        {
+            _ = Harmony.CreateAndPatchAll(typeof(BetterColorPicker));
```

Comment:
I don't use discards outside of ref/out parameters. This in particular will light up the unused variable rule, and will prevent people on older versions of VS (or other IDEs) from compiling.

Reply:
Remove the discard.  Used to them because of Python.